ambari-agent/src/main/python/ambari_agent/hostname.py

```diff
@@ -131,9 +131,9 @@ def server_hostnames(config):
 
 
 def main(argv=None):
-  print hostname()
-  print public_hostname()
-  print server_hostname()
+  print(hostname())
+  print(public_hostname())
+  print(server_hostname())
```

Comment:
server_hostname => server_hostnames
![image](https://github.com/apache/ambari/assets/12639125/82bebfa7-3916-4eb3-8f19-3552fae0b790)

Reply:
thanks for finding this, I will fix it later.